src/enums/DatatypeEnum.php

```diff
@@ -8,8 +9,7 @@
  *
  * todo: needs a basic parent enum class for error handling.
  */
-class DatatypeEnum
-{
+class DatatypeEnum {
```

Comment:
Now I'm confused: thought these brackets have to be on the next line? (e.g. https://www.php-fig.org/psr/psr-2/)